### ISSUE TITLE
Some help dialog strings remain un-translatable

### ISSUE THREAD

**Issue by phet-steele** [phetsims/balloons-and-static-electricity#397]
The BASE specific strings in the keyboard help dialog are not appearing to be translatable (using `?stringTest=double`). The strings on the left-hand side are not doubled:

<img width="1427" alt="screen shot 2018-04-12 at 10 28 43 am" src="https://user-images.githubusercontent.com/5863899/38691160-3daebbde-3e3d-11e8-94a3-85e25741b405.png">

Seen on macOS 10.13.3 Chrome. For phetsims/QA/issues/112.

<details>
<summary>Troubleshooting Information</summary>

URL: https://www.colorado.edu/physics/phet/dev/html/balloons-and-static-electricity/1.4.0-dev.21/phet/balloons-and-static-electricity_all_phet_debug.html?accessibility&stringTest=double
Version: 1.4.0-dev.21 2018-04-04 18:25:56 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.181 Safari/537.36
Language: en-US
Window: 1920x1013
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 32 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"928741cf","branch":"master"},"axon":{"sha":"ceb3cc00","branch":"master"},"balloons-and-static-electricity":{"sha":"f4f3cc64","branch":"master"},"brand":{"sha":"89d28f63","branch":"master"},"chipper":{"sha":"bc7c5d1f","branch":"master"},"dot":{"sha":"e5489d18","branch":"master"},"joist":{"sha":"2cf256a9","branch":"master"},"kite":{"sha":"b6071478","branch":"master"},"phet-core":{"sha":"d8804eee","branch":"master"},"phet-io":{"sha":"4bbddfaa","branch":"master"},"phet-io-website":{"sha":"b6c2b0f5","branch":"master"},"phet-io-wrapper-classroom-activity":{"sha":"1ecd81a1","branch":"master"},"phet-io-wrapper-lab-book":{"sha":"b1c5562c","branch":"master"},"phet-io-wrappers":{"sha":"025cc23d","branch":"master"},"phetcommon":{"sha":"de004c6e","branch":"master"},"query-string-machine":{"sha":"c379f1db","branch":"master"},"scenery":{"sha":"4a4afee2","branch":"master"},"scenery-phet":{"sha":"e68ebc03","branch":"master"},"sherpa":{"sha":"bb14a03a","branch":"master"},"sun":{"sha":"8a187c9f","branch":"master"},"tandem":{"sha":"f8147666","branch":"master"},"vibe":{"sha":"2dc20be7","branch":"master"}}
</details>

**Comment by jessegreenberg** [comment]
Thanks very much @phet-steele, I forgot about these.  Strings have been added to translatable strings. Now the dialog is too big for dev bounds.
![capture](https://user-images.githubusercontent.com/6396244/38704509-d2feed8c-3e63-11e8-8cd2-70a9dcf890da.PNG)

**Comment by jessegreenberg** [comment]
Done with the above commits. @phet-steele closing since this one was straight forward (feel free to double check me if you think that is best)
